### ISSUE TITLE
A11Y: ClickableTile is not getting selected when using keyboard

### ISSUE THREAD

**Issue by ashpc** [carbon-design-system/carbon-addons-cloud#20]
## Detailed description

When using keyboard to navigate through the page that has ClickableTile, the tiles are not getting highlighted. 

> Is this issue related to a specific component?

ClickableTile

> What did you expect to happen? What happened instead? What would you like to see changed?

When I use tab(keyboard) on a page that has a ClickableTile, then the tiles should also gain focus. But this is not happening and the focus moves to the next element.

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

"carbon-addons-beta-react": "1.3.2",
"carbon-addons-bluemix": "0.2.0",
"carbon-components": "8.1.12",
"carbon-components-react": "5.0.6",
"carbon-icons": "6.1.0"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

This is to resolve any A11Y issues we have in our UI, it would be great if we can get it fixed asap.

## Steps to reproduce the issue

1. Use tab(keyboard) to navigate through a page that has ClickableTile and you will see that the tile is not getting selected.

## Add labels
bug

**Comment by tw15egan** [comment]
The tiles will gain focus when supplied with an `href`. Since the `ClickableTile`is used for navigation, they are wrapped in anchor elements. Anchor elements will not receive keyboard focus if they do not have a `href` supplied. Can you confirm this is still a bug when an `href` is provided?

**Comment by ashpc** [comment]
@tw15egan looks like I was not passing href but using the handleClick to do what we needed. I will use href and see if it is still having issues. Thanks a lot for a quick response.

**Comment by joshblack** [comment]
Going to close this due to inactivity, feel free to re-open if it's still an issue!
